-------------------------------------------------------------------------------
Test set: myproperty.v1._controller.PersonControllerIT
-------------------------------------------------------------------------------
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.601 sec <<< FAILURE! - in myproperty.v1._controller.PersonControllerIT
testGetPeople(myproperty.v1._controller.PersonControllerIT)  Time elapsed: 0.601 sec  <<< FAILURE!
java.lang.AssertionError: Something Went Wrong 
	at org.junit.Assert.fail(Assert.java:88)
	at myproperty.v1._controller.PersonControllerIT.testGetPeople(PersonControllerIT.java:57)

